junit-jupiter-params/src/main/java/org/junit/jupiter/params/ExpressionLanguageAdapter.java

```diff
@@ -0,0 +1,9 @@
+
+package org.junit.jupiter.params;
+
+public interface ExpressionLanguageAdapter {
+
+	void compile(String template);
+
+	void format(Object scope, StringBuffer result);
```

Comment:
I think we should split this into two to allow (not necessarily in this PR) registering a global default expression language that would support instantiating the `ExpressionLanguageAdapter` at most once and also provide a hook to clean up resources by extending `AutoCloseable`.

```suggestion
	Expression compile(String template);
```

```java
interface Expression {
	void evaluate(Map<String, Object> context, Appendable result);
}
```